packages/Liquid-UI.package/LQPollDraftBuilder.class/instance/removeWhitespacesFrom..st

```diff
@@ -0,0 +1,7 @@
+modifying
+removeWhitespacesFrom: aPollDraft
```

Comment:
This method does not belong the UI directly. Such logic should either be located in a helper class (utility-approach) or in the LQPollDraft-class itself (OO-approach). This makes testing much simpler and decouples the UI. Have a look at the LQChoiceProcessor.

Reply:
As this method is very short and is not complex, we would prefer the OO-approach to have less class-cluttering.